banking/klarna_kosma_integration/doctype/bank_reconciliation_tool_beta/bank_reconciliation_tool_beta.py

```diff
@@ -1456,4 +1456,7 @@ def bank_query(doctype: str, txt: str, searchfield: str, start: int, page_len: i
 		limit_start=start,
 		limit_page_length=page_len,
 	)
+	results = list(dict.fromkeys(results))
+	results.sort()
```

Comment:
```suggestion
		group_by="bank",
		order_by="bank"
	)
```

Let's leave the hard work to the database